packages/api/test/util/describeE2E.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017-2019 @polkadot/api authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { WsProvider } from '@polkadot/rpc-provider';
+
+import { ApiPromise, ApiRx } from '../../src';
+
+const WS_ENDPOINTS = {
+  'local': 'ws://127.0.0.1:9944/',
+  'substrate-master': 'ws://127.0.0.1:9945/',
+  'substrate-1.0': 'ws://127.0.0.1:9946/',
+  'substrate-2.0': 'ws://127.0.0.1:9947/',
+  'polkadot-master': 'ws://127.0.0.1:9948/',
+  'polkadot-alexander': 'ws://127.0.0.1:9949/',
+  'remote-polkadot-alexander': 'wss://poc3-rpc.polkadot.io/',
+  'remote-substrate-1.0': 'wss://substrate-rpc.parity.io/'
+};
```

Comment:
I already added them to the jest globals, so they should be available with `global.ws_local`, `global.ws_polkadot_master`etc https://github.com/polkadot-js/api/blob/master/jest.config.js#L23

If we only need them here, I should remove them from the config.

Reply:
I took the same names as in the docker-compose files, (with `remote-*` for remote ones), but I can take your naming in jest config if you want

Reply:
Nah, dash is fine. I have no strong feelings about the naming